### ISSUE TITLE
Edit profile fields look wonky

### ISSUE THREAD

**Issue by stephaniesimms** [DMPRoadmap/roadmap#230]
Edit profile fields display in a weird, misaligned manner (screenshot below). Organisation drop-down is clearly shifted to the right. Other fields look oddly spaced too.

![screen shot 2017-04-10 at 8 05 53 pm](https://cloud.githubusercontent.com/assets/12703638/24891234/5b896e68-1e29-11e7-9243-c958a7ee8bf1.png)

**Comment by briri** [comment]
This should be fixed when we replace select2 and replace the use of html tables to format these forms. 
I tried updating some of the css on the `select2-container` but nothing seemed to make it budge.

**Comment by stephaniesimms** [comment]
ok @briri - I'll remove it from MVP-v0.3 and tag w/MVP.

**Comment by briri** [comment]
related to #125

**Comment by sjDCC** [comment]
Fixed on both staging and test. Closing out
